reference/constraints/Timezone.rst

```diff
@@ -85,8 +86,34 @@ string meant to contain a timezone identifier (e.g. ``America/New_York``):
 Options
 -------
 
+countryCode
+~~~~~~~~~~~
+
+**type**: ``string`` **default**: ``null``
+
+If the ``zone`` option is set to ``\DateTimeZone::PER_COUNTRY``, this option
+restricts the valid timezone identifiers to the ones that belong to the given
+country.
+
+The value of this option must be a valid `ISO 3166-1 alpha-2`_ country code
+(e.g. ``CN`` for China).
+
 .. include:: /reference/constraints/_groups-option.rst.inc
 
+intlCompatible
+~~~~~~~~~~~~~~
+
+**type**: ``boolean`` **default**: ``false``
+
+This constraint considers valid both the `PHP timezone identifiers`_ and the
+:ref:`ICU timezones <component-inlt-timezones>` provided by Symfony's
+:doc:`Intl component </components/intl>`
+
+However, the timezones provided by the Intl component can be different from the
+timezones provided by PHP's Intl extension because the ICU versions can be
+different. If this option is set to ``true``, this constraint also considers
+valid any PHP ``\IntlTimeZone`` value.
```

Comment:
`... considers valid any value for PHPs ``\IntlTimeZone::createTimeZone()`` method` is perhaps more explicit